packages/ListBox/src/components/Option/Option.styles.js

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+const blueSelected = "#e5f1fe";
+
+export const OptionStyled = styled.li`
+  padding: 4px;
+  margin-bottom: 4px;
+  font-size: 13px;
+  border-radius: 3px;
+
```

Comment:
`cursor:pointer`, or is there a reason you leave it out?

Reply:
thank you for the input Tristan, I added a small disclaimer at the top of this PR that I'm not focusing on styling at this moment, it will be done in a different PR.